Sources/SwiftDocC/Indexing/Navigator/NavigatorItem.swift

```diff
@@ -90,98 +88,95 @@ public final class NavigatorItem: Serializable, Codable, Equatable, CustomString
         self.availabilityID = availabilityID
         self.icon = icon
     }
-    
+
     // MARK: - Serialization and Deserialization
-    
+
     /**
      Initialize a `NavigatorItem` using raw data.
-     
+
      - Parameters rawValue: The `Data` from which the instance should be deserialized from.
      */
     required public init?(rawValue: Data) {
         let data = rawValue
-        
+
         var cursor: Int = 0
         var length: Int = 0
-        
+
         length = MemoryLayout<UInt8>.stride
         self.pageType = unpackedValueFromData(data[cursor..<cursor + length])
         cursor += length
-        
+
         self.languageID = unpackedValueFromData(data[cursor..<cursor + length])
         cursor += length
-        
+
         length = MemoryLayout<UInt64>.stride
         self.platformMask = unpackedValueFromData(data[cursor..<cursor + length])
         cursor += length
-            
+
         self.availabilityID = unpackedValueFromData(data[cursor..<cursor + length])
         cursor += length
-        
+
         let titleLength: UInt64 = unpackedValueFromData(data[cursor..<cursor + length])
         cursor += length
-        
+
         let pathLength: UInt64 = unpackedValueFromData(data[cursor..<cursor + length])
         cursor += length
-        
+
         let titleData = data[cursor..<cursor + Int(titleLength)]
         cursor += Int(titleLength)
         self.title = String(data: titleData, encoding: .utf8)!
-        
+
         let pathData = data[cursor..<cursor + Int(pathLength)]
         self.path = String(data: pathData, encoding: .utf8)!
-        
-        assert(cursor+Int(pathLength) == data.count)
+
+        assert(cursor + Int(pathLength) == data.count)
     }
 
     /// Returns the `Data` representation of the current `NavigatorItem` instance.
     public var rawValue: Data {
         var data = Data()
-        
+
         data.append(packedDataFromValue(pageType))
         data.append(packedDataFromValue(languageID))
         data.append(packedDataFromValue(platformMask))
         data.append(packedDataFromValue(availabilityID))
         data.append(packedDataFromValue(UInt64(title.utf8.count)))
         data.append(packedDataFromValue(UInt64(path.utf8.count)))
-        
+
         data.append(Data(title.utf8))
         data.append(Data(path.utf8))
-        
+
         return data
     }
-    
+
     // MARK: - Equatable
-    
+
     public static func == (lhs: NavigatorItem, rhs: NavigatorItem) -> Bool {
-        return lhs.pageType == rhs.pageType &&
-            lhs.languageID == rhs.languageID &&
-            lhs.title == rhs.title &&
-            lhs.platformMask == rhs.platformMask &&
-            lhs.availabilityID == rhs.availabilityID
+        return lhs.pageType == rhs.pageType && lhs.languageID == rhs.languageID && lhs.title == rhs.title && lhs.platformMask == rhs.platformMask
```

Comment:
I thought [`"respectsExistingLineBreaks" : true`](https://github.com/daniel-grumberg/swift-docc/pull/1/files#diff-71a9f0678f9477617270d9ab4b83b53559efb4e883fbe55ee8ee42195611587cR13) would have preserved this 🤔